rfcs/text/0000-project-completion-stage.md

```diff
@@ -0,0 +1,74 @@
+
+**More thorough data validation:**
+The downstream services of the DCP have the potential to uncover metadata and data errors. Viewing the version-complete project as the user will view it can also identify errors. A solution to this is a setting on projects allowing signed-in DCP members to access the data in order to perform a final ‘sanity check’.
+
+**Support data contributors (possibly 2nd phase):**
+Allowing data contributors to view the final product in their collaboration with the DCP before it is released adds yet another validation step (after all, who knows their data better?) and is a valuable step in good faith data sharing. The inclusion of a data lifecycle tracker on the project page keeps the contributor up-to-date on their submission and ensures a transparent process.
+
+### User Stories
+
+As a consumer of HCA data, I would assume that a project is version-complete if it is displayed on the DataBrowser or searchable in the DataStore and will not check back for more data analysis products.
+
+As a consumer of HCA data, I will be more confident in using raw data in my own analysis if they have been successfully run on the DCP-hosted pipelines.
+
+As a consumer of HCA data, I will be more confident in using metadata, raw data, or analysis products from a project if all data and data provenance are available and easily referenced.
+
+As a consumer of HCA data, I do not want to see Integration Test projects on the DataBrowser or DataStore, especially if I have a notification set up for new projects as these will trigger my notification but don’t actually represent new data.
+
+As a member of the DCP, I cannot do a comprehensive final check on data on the DataBrowser or DataStore unless I see it as the data consumer will see it.
+
+As a member of the DCP, I do not want inaccurate metadata or products of incorrect data analysis being shown to the public. 
+Examples of inaccuracies that were/are publicly accessible:
+- Optimus results on Smart-seq2 data ([#2314](https://github.com/HumanCellAtlas/data-store/issues/2314))
+- ‘Homo sapiens’ vs ‘homo sapiens’ ([ZD ticket 158](https://humancellatlas.zendesk.com/agent/tickets/158))
+- Project ingested at the wrong schema version ([#2421](https://github.com/HumanCellAtlas/data-store/issues/2421))
+- Project Metadata tsv download threw an error
+- Contributor name misspelled ("Tom Mithcell")
+
+As a contributor whose name is associated with the project, I want to be allowed time to review the full details of my contribution as for accuracy before it becomes publicly available. I want this review to be through the vantage of a potential user. (possibly 2nd phase)
```

Comment:
Like with contributor review of analysis results we need to make sure if such a mechanism is introduced that it doesn't establish the expectation that all named scientists in a project get to actively okay a project submission. The current system trusts the person directly interacting with the wrangler to be able to give permission for all named scientists and I expect that will continue when that contributor is directly interacting with the system.

Within ingest we have a long term vision which we haven't yet realised which would allow different participants in the project lifecycle being able to enter the data or metadata they are responsible for (tissue acquisition potentially being different from assay details) but even if that is possible it is still most likely that for most submissions only one person will interact with us and that should be sufficient. Hopefully, our authentication and authorization system will ultimately allow multiple people to log in and view but it shouldn't be required

Reply:
Excellent point. I have added that clarification in b175eb3.

Reply:
Resolving this thread, please renew comments if I can address things further in the RFC.